co2eq/countries.json

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
I would rename this file to `hotelpercountry.json` and have the following structure:
```json
{
  "AF": { "footprintKilogramsPerKilowatthours": 64.1 },
  ...
}
```

Rationale:
- This file only has information about hotels (not about countries in general)
- The unit should be explicit from the naming to avoid any mistakes (feel free to adjust the name)
- The `name` key in countries is not needed so it can discarded (and countries without footprints can be discarded)

Reply:
I tried to have a general structure on purpose so this file could also be used for other country related information in the future, as I imagine it is likely that other kinds of information would depend on countries.

The presence of countries that don't have footprint has the following reasons:
- It allows to check if the country code provided by "activity" is an existing one before running the model. Otherwise the model would return the default value even for invalid values of the country code variable.
- The file is already ready with all the existing ISO2 country codes to be able to hold a new kind of country related information for a different model if needed in the future.
- When we will find new data to complete the model, we will just have to add the footprint value and not write the country code anymore.
- Even without additionnal data, it would be possible to improve the model by not returning a unique default value, but different default values depending for example on the area, like a different value per continent. That could be done for example by adding a key for a more general area in each of the country without a footprint value. Keeping them in the file is then useful.
At this point, I think that deleting those countries would be a waist of time since it is likely we will need to add them again later on anyway.

I added a name key because I thought it could be usefull in the futur. It also helps the programmers who are going to deal with this file to add more data or update the existing data to very easily know wich code corresponds to which country since it is usually not obvious. But it is possible that the space it takes is too much compared to that benefit.

It is very true that a more explicit unit would be helpful. I will change that. The actual unit is kilogram per night of stay and per room by the way.

Reply:
Hello,
Thanks for the write-up, now I understand better.
I think it is indeed a good idea to have a central file. In order to proceed, I suggest renaming the key to make the unit clearer and I believe "footprint" shouldn't be abbreviated (for clarity).
I would remove the name though as it makes the file grow in space, and the use of it is unclear at this stage (I don't think being a hint for the developer justifies having it in the file. It could be a comment but comments are not possible in JSON).
- Olivier